### DISCUSSION TITLE
Custom Recipes

### DISCUSSION THREAD

**Discussion by MCTyler** [Zrips/CMI#90] [Feature Requests]
Is there any way I can have CMI over ride Minecraft default Recipes with Custom Recipes? Also, using a custom Furnace recipe, I would like to add some cost to it, (IE EXP = 5 for Smelting Steel to Iron)

My example of this is using Steel Ingots to craft Steel Armor but using the same recipe as Iron Armor only with the newly smelted steel ingots and some enchantments..

**Comment by mrfloris** [comment]
Having further controls over the basic recipes, as well as overriding them, and extending custom recipes with NBT data items would be a great addition. When we go into the recipe editor from CMI, it would be nice if we could set a title to a (custom) recipe. And block existing recipes from Minecraft one way or another. And to have a custom recipe accept amounts and custom items as well. 
(Change title, accept quantity for input / output, and nbt items for input with custom titles/lore/etc. )

In the last 12 months I've had my own wishes for extending the vanilla and CMI Custom Recipe features, and had a couple of discussions with Zrips about it. And I have seen a handful to a dozen requests of various sorts about the recipe feature doing more than it can. I think a request to extend this is still desirable. Maybe through a module that when turned on will download a library remotely once like geo/lang/paperlib stuff. RP/RPG, hardcore and survival servers can really benefit from this.
